README.md

```diff
@@ -132,12 +132,14 @@ Enter this into the `slack_member_id` field of your alert configuration.
 Clade assignments and other QC metadata output by Nextclade are currently cached in `nextclade.tsv` in the S3 bucket and only incremental additions for the new sequences are performed during the daily ingests.
 Whenever the underlying nextclade dataset (reference tree, QC rules) and/or nextclade software are updated, it is necessary to perform a full update of `nextclade.tsv`, rerunning for all of the GISAID and GenBank sequences all over again, to account for changes in the data and Nextclade algorithms.
 
-In order to tell ingest to not use the cached `nextclade.tsv` and instead perform a full rerun, you need to add an (empty) touchfile to the s3 bucket:
+In order to tell ingest to not use the cached `nextclade.tsv`/`aligned.fasta` and instead perform a full rerun, you need to add an (empty) touchfile to the s3 bucket:
 
 ```bash
 touch nextclade.tsv.zst.renew
 aws s3 cp nextclade.tsv.zst.renew s3://nextstrain-ncov-private/nextclade.tsv.zst.renew
+aws s3 cp nextclade.tsv.zst.renew s3://nextstrain-ncov-private/aligned.fasta.zst.renew
 aws s3 cp nextclade.tsv.zst.renew s3://nextstrain-data/files/ncov/open/nextclade.tsv.zst.renew
+aws s3 cp nextclade.tsv.zst.renew s3://nextstrain-data/files/ncov/open/aligned.fasta.zst.renew
```

Comment:
Since I saw this in passing, I'll note the local empty file can be omitted by changing the commands to, e.g.:

```
aws s3 cp - s3://nextstrain-ncov-private/aligned.fasta.zst.renew < /dev/null
```

Reply:
Ah nice - good spot :)